paprika/__init__.py

```diff
@@ -29,3 +29,7 @@
                     dest='loglevel',
                     const=logging.INFO)
 # args = parser.parse_args()
+
+### NMH I want `import paprika` to import everything ...
+#import align,conformation_prep,restraints,solvate,version
+
```

Comment:
Go into `__init__.py` and add a line that says 
```
__all__ = ["align", "build", "restraints", "simulate"]
```

Reply:
AFAIK, this line (which is absent here) controls what is imported with the wildcard `*`.

Reply:
done